extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1665 @@
+        "EXT_feature_metadata": {
+          "featureTextures": ["heatLossTexture"]
+        }
+      }
+    }
+  ]
+}
+```
+
+
+`texture` is a glTF [`textureInfo`](../../../../../specification/2.0/schema/textureInfo.schema.json) object. `texCoord` refers to the texture coordinate set of the referring primitive. `channels` is a string matching the pattern `"^[rgba]{1,4}$"` that specifies which texture channels store property values.
+
+### Statistics
+
+Statistics provide aggregate information about features in the model. Statistics are provided on a per-class basis.
+
+* `count` is the number of features that conform to the class
+* `properties` contains statistics about property values
+
+Properties have the following built-in statistics:
+
+Name|Description|Type
+--|--|--
+`min`|The minimum property value|Numeric types or fixed-length arrays of numeric types
+`max`|The maximum property value|...
+`mean`|The arithmetic mean of the property values|...
+`median`|The median of the property values|...
+`standardDeviation`|The standard deviation of the property values|...
+`variance`|The variance of the property values|...
+`sum`|The sum of the property values|...
+`occurrences`|Number of enum occurrences|Enums or fixed-length arrays of enums
+
+Model authors may define their own additional statistics, like `mode` below.
+
+```jsonc
+{
+  "extensions": {
+    "3DTILES_metadata": {
+      "schema": {
+        "classes": {
+          "building": {
+            "properties": {
+              "height": {
+                "type": "FLOAT32"
+              },
+              "owners": {
+                "type": "ARRAY",
+                "componentType": "STRING"
+              },
+              "buildingType": {
+                "type": "ENUM",
+                "enumType": "buildingType"
+              }
+            }
+          }
+        },
+        "enums": {
+          "buildingType": {
+            "valueType": "UINT8",
+            "values": [
+              {
+                "name": "Residential",
+                "value": 0
+              },
+              {
+                "name": "Commercial",
+                "value": 1
+              },
+              {
+                "name": "Hospital",
+                "value": 2
+              },
+              {
+                "name": "Other",
+                "value": 3
+              }
+            ]
+          }
+        }
+      },
+      "statistics": {
+        "classes": {
+          "building": {
+            "count": 100,
+            "properties": {
+              "height": {
+                "min": 3.9,
+                "max": 341.7,
+                "mode": 5.0
+              },
+              "buildingType": {
+                "occurrences": {
+                  "Residential": 70,
+                  "Commercial": 28,
+                  "Hospital": 2
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Examples
```

Comment:
A good supplement to this table that will help the reader build their mental model would be a diagram that shows the breath of granularity from per-texel to per-vertex to per-triangle to per-node, etc.

That would be helpful early in the spec.